packages/hoppscotch-common/src/components/lenses/renderers/HTMLLensRenderer.vue

```diff
@@ -128,6 +128,7 @@ useCodemirror(
     linter: null,
     completer: null,
     environmentHighlights: true,
+    predefinedVariablesHighlights: true,
```

Comment:
I can't think of a use case where we'd need to highlight pre-defined variable references in responses since they'd already be unwrapped during request execution; keeping it as is for now since it was specified for environment variable highlights.